packages/cbioportal-clinical-timeline/src/TimelineStore.ts

```diff
@@ -49,6 +51,15 @@ export class TimelineStore {
         setTimeout(this.setScroll.bind(this), 10);
     }
 
+    setGroupByOption(value: string) {
+        this.groupByOption = value;
+        console.info(this.groupByOption);
+    }
+
+    getGroupByOption() {
```

Comment:
i would just reference this directly.   not worth wrapping in a method.

Reply:
fixed